shared/utility/math/coordinates.hpp

```diff
@@ -51,19 +51,23 @@ namespace utility::math::coordinates {
 
     template <typename T, typename U = typename T::Scalar>
     inline Eigen::Matrix<U, 3, 1> cartesianToSpherical(const Eigen::MatrixBase<T>& cartesianCoordinates) {
-        U x = cartesianCoordinates.x();
-        U y = cartesianCoordinates.y();
-        U z = cartesianCoordinates.z();
+        const U x = cartesianCoordinates.x();
+        const U y = cartesianCoordinates.y();
+        const U z = cartesianCoordinates.z();
         Eigen::Matrix<U, 3, 1> result;
 
-        result.x() = std::sqrt(x * x + y * y + z * z);  // r
-        result.y() = std::atan2(y, x);                  // theta
-        if (result.x() == static_cast<U>(0)) {
-            result.z() = static_cast<U>(0);
-        }
-        else {
-            result.z() = std::asin(z / (result.x()));  // phi
-        }
+        result.x() = std::sqrt(x * x + y * y + z * z);                                                     // r
+        result.y() = std::atan2(y, x);                                                                     // theta
+        result.z() = (result.x() == static_cast<U>(0)) ? static_cast<U>(0) : std::asin(z / (result.x()));  // phi
+
+        return result;
+    }
+
+    template <typename T, typename U = typename T::Scalar>
+    inline Eigen::Matrix<U, 3, 1> inverseDistanceCartesianToSpherical(
```

Comment:
```suggestion
    [[nodiscard]] inline Eigen::Matrix<U, 3, 1> inverseDistanceCartesianToSpherical(
```

Reply:
I would rename this function to `cartesianToInverseDistanceSpherical`
Otherwise it implies the cartesian has inverse distance not the spherical